src/pudl/metadata/fields.py

```diff
@@ -2520,6 +2525,11 @@
         "description": "Original reported energy interchange between adjacent balancing authorities.",
         "unit": "MWh",
     },
+    "interconnect_code_eia": {
+        "type": "string",
+        "description": "EIA interconnect code.",
+        "constraints": {"enum": {"EASTERN", "WESTERN", "ERCOT"}},
+    },
```

Comment:
In most cases outside of canonical abbreviations (e.g. for states) we've defaulted to using `snake_case` for code values, since otherwise you end up with a mix of cases so it seems a little funny for just these to be all caps.

Reply:
Fixed